manifests/apache/vhost.pp

```diff
@@ -99,22 +99,24 @@
   Boolean $ldap_require_group               = $puppetboard::params::ldap_require_group,
   Optional[String] $ldap_require_group_dn   = undef,
   Hash $custom_apache_parameters            = {},
+  Hash $wsgi_daemon_process_options_extra   = {},
 ) inherits ::puppetboard::params {
-
   $docroot = "${basedir}/puppetboard"
 
   $wsgi_script_aliases = {
     "${wsgi_alias}" => "${docroot}/wsgi.py",
   }
 
-  $wsgi_daemon_process_options = {
+  $wsgi_daemon_process_options_default = {
     threads => $threads,
     group   => $group,
     user    => $user,
   }
+  $wsgi_daemon_process_options = merge($wsgi_daemon_process_options_default, 
+                                       $wsgi_daemon_process_options_extra)
```

Comment:
This looks like an unrelated bugfix to modulesync. I think you can also use `+`
```suggestion
  $wsgi_daemon_process_options_default = {
    threads => $threads,
    group   => $group,
    user    => $user,
  } + $wsgi_daemon_process_options_extra
```

I'll leave it up to you to decide if it's more or less readable.